src/cards/repel.ts

```diff
@@ -1,18 +1,19 @@
 import { getCurrentTargets, Spell } from './index';
 import { CardCategory } from '../types/commonTypes';
 import { playDefaultSpellSFX } from './cardUtils';
-import { forcePush } from './push';
+import { forcePush, pushId } from './push';
 import { CardRarity, probabilityMap } from '../types/commonTypes';
 
 export const id = 'repel';
 export const velocityStartMagnitude = 10;
 const spell: Spell = {
   card: {
     id,
+    requires: [pushId],
     category: CardCategory.Movement,
     supportQuantity: true,
     sfx: 'push',
-    manaCost: 10,
+    manaCost: 20,
```

Comment:
You mentioned considering reverting the mana change.  Maybe it should be 15?  I like keeping movement spells cheap, but if it's 10 then there's literally no reason to ever use push again.